packages/client/modules/teamDashboard/components/TeamDashInsights/TopRetroTemplatesCard.tsx

```diff
@@ -36,20 +38,21 @@ const TopRetroTemplatesCard = (props: Props) => {
     <TeamInsightsCard
       teamInsightsRef={insights}
       title='Top Templates'
-      tooltip='The most frequently used retrospective templates on your team'
+      tooltip='The most frequently used retrospective templates on your team for the last 12 months'
```

Comment:
+1 this is a fairly long tooltip. Maybe: `The most used retro templates on your team in the last 12 months`?